.github/workflows/ci-nextjs.yml

```diff
@@ -19,10 +19,64 @@ concurrency:
 
 jobs:
 
-  build-deploy-next-app:
-      name: Build & Deploy
-      uses: ./.github/workflows/build-deploy.yml
+  # Commenting these out for now as there's currently an issue with Next.
```

Comment:
Do we have a ticket for this?

Reply:
I'm not really sure we can do anything apart from wait for a new version of Next to be honest.

I was hoping this will indirectly be done when we add renovate. Personally, given the amount of time i've spent already trying to solve this issue, our time is better spent testing Lit 3, and seeing if that, in combination with removing our `resolutions` hack will solve the issue.